lib/db.js

```diff
@@ -215,6 +217,31 @@ class LimitDBRedis extends EventEmitter {
       });
     }
 
+    if (bucketKeyConfig.skip_n_calls > 0) {
```

Comment:
We should probably make a reference to the DR

Reply:
Agree! as an outcome. But since the name might change :D

Reply:
I was thinking of adding a comment with the DR or something similar.

Reply:
Done!